BoltsTest/src/bolts/CancellationTest.java

```diff
@@ -45,4 +46,67 @@ public void testTokenThrowsWhenCancelled() {
       // Do nothing
     }
   }
+
+  public void testTokenCallsRegisteredActionWhenCancelled() {
+    CancellationTokenSource cts = new CancellationTokenSource();
+    CancellationToken token = cts.getToken();
+    final Capture<Object> result = new Capture<>();
+
+    token.register(new Runnable() {
+      @Override
+      public void run() {
+        result.set("Run!");
```

Comment:
When the cancellation runnable fires? I think the token should take care of that